FreeSurfer/FreeSurferPipelineNHP.sh

```diff
@@ -0,0 +1,1219 @@
+#!/bin/bash
```

Comment:
One question would be how severe is the diff from the human pipeline for the base pipeline script, as this looks to be based on a relatively recent version of the human pipeline script?  Another is whether longitudinal mode is supported for NHP (or if it should be supported)?